tests/test_entities.py

```diff
@@ -162,225 +168,347 @@
+            assert elements_equal(
+                s.root, ElementTree.fromstring(self.modcont_step_placements_xml)
+            )
 
 
 class TestArtifacts(TestEntities):
     root_artifact_xml = generic_artifact_xml.format(url=url)
 
     def test_input_artifact_list(self):
-        a = Artifact(uri=self.lims.get_uri('artifacts', 'a1'), lims=self.lims)
-        with patch('requests.Session.get', return_value=Mock(content=self.root_artifact_xml, status_code=200)):
+        a = Artifact(uri=self.lims.get_uri("artifacts", "a1"), lims=self.lims)
+        with patch(
+            "requests.Session.get",
+            return_value=Mock(content=self.root_artifact_xml, status_code=200),
+        ):
             assert a.input_artifact_list() == []
 
     def test_workflow_stages_and_statuses(self):
-        a = Artifact(uri=self.lims.get_uri('artifacts', 'a1'), lims=self.lims)
+        a = Artifact(uri=self.lims.get_uri("artifacts", "a1"), lims=self.lims)
         expected_wf_stage = [
-            (Stage(self.lims, uri=url + '/api/v2/configuration/workflows/1/stages/2'), 'QUEUED', 'Test workflow s2'),
-            (Stage(self.lims, uri=url + '/api/v2/configuration/workflows/1/stages/1'), 'COMPLETE', 'Test workflow s1')
+            (
+                Stage(
+                    self.lims, uri=url + "/api/v2/configuration/workflows/1/stages/2"
+                ),
+                "QUEUED",
+                "Test workflow s2",
+            ),
+            (
+                Stage(
+                    self.lims, uri=url + "/api/v2/configuration/workflows/1/stages/1"
+                ),
+                "COMPLETE",
+                "Test workflow s1",
+            ),
         ]
-        with patch('requests.Session.get', return_value=Mock(content=self.root_artifact_xml, status_code=200)):
+        with patch(
+            "requests.Session.get",
+            return_value=Mock(content=self.root_artifact_xml, status_code=200),
+        ):
             assert a.workflow_stages_and_statuses == expected_wf_stage
 
 
 class TestReagentKits(TestEntities):
-    url = 'http://testgenologics.com:4040'
+    url = "http://testgenologics.com:4040"
     reagentkit_xml = generic_reagentkit_xml.format(url=url)
 
     def test_parse_entity(self):
-        r = ReagentKit(uri=self.lims.get_uri('reagentkits', 'r1'), lims=self.lims)
-        with patch('requests.Session.get', return_value=Mock(content=self.reagentkit_xml, status_code=200)):
-            assert r.name == 'regaentkitname'
-            assert r.supplier == 'reagentProvider'
-            assert r.website == 'www.reagentprovider.com'
-            assert r.archived == False
+        r = ReagentKit(uri=self.lims.get_uri("reagentkits", "r1"), lims=self.lims)
+        with patch(
+            "requests.Session.get",
+            return_value=Mock(content=self.reagentkit_xml, status_code=200),
+        ):
+            assert r.name == "regaentkitname"
```

Comment:
Tempting to want to change this, but I don't know where it origins from

Reply:
[afe846e](https://github.com/SciLifeLab/genologics/pull/244/commits/afe846e3f18cabc4fe831cf8f63b7adcad8970f7)

I just replace `regaent`with reagent, 3 hits repo wide.